modules/svg/image_loader_svg.cpp

```diff
@@ -173,9 +173,17 @@ void ImageLoaderSVG::get_recognized_extensions(List<String> *p_extensions) const
 }
 
 Error ImageLoaderSVG::load_image(Ref<Image> p_image, Ref<FileAccess> p_fileaccess, BitField<ImageFormatLoader::LoaderFlags> p_flags, float p_scale) {
-	String svg = p_fileaccess->get_as_utf8_string();
+	const uint64_t len = p_fileaccess->get_length() - p_fileaccess->get_position();
+	Vector<uint8_t> buffer;
```

Comment:
I don't think it's possible to have length < -1 or >= p_fileaccess->get_length(), so we don't need a err index check but..

Reply:
However is missing is_null checks on the inputs.

Reply:
Which input do you mean? Null checks for `p_image` and `p_fileaccess` are done at `ImageLoader::load_image()` level.